Testscripts/Linux/XDPDumpSetup.sh

```diff
@@ -110,6 +112,25 @@ function Run_XDPDump {
     fi
 }
 
+# Check if kernel supports XDP or not
+function check_xdp_support {
+    if [ -z "${1}" -o -z "${2}" ]; then
+        LogErr "ERROR: must provide install ip and NIC Name to Run_XDPDump"
+        SetTestStateAborted
+        exit 1
```

Comment:
If supporting as utility function then return. Don't exit and set state.

Reply:
please refer to previous discussion.